### ISSUE TITLE
An Issue with the .bat file

### ISSUE THREAD

**Issue by MysticMCO** [ntdevlabs/tiny11builder#24]
Whenever I launch the .bat file and press enter the thing just force closes me and i cannot type the ISO mounted drive letter.

**Comment by MysticMCO** [comment]
I apologize but i am stupid and don't know anything

**Comment by clorophilla** [comment]
Try launching the .bat file from Windows Terminal (if you're on 11) or CMD/Powershell (if on 10) as administrator, you'll find it pressing Windows and X. Then you'll see which error the script will report.

**Comment by MysticMCO** [comment]
Thank you. Now it at least worked but now it's showing me this
![image](https://user-images.githubusercontent.com/111479268/224275161-9f230fdc-f72b-4d3b-9312-1c642e783341.png)


for reference this is the windows DVD drive file

![lmao](https://user-images.githubusercontent.com/111479268/224275784-298d1c51-e97d-492c-b177-f8d739bcb6c7.png)
![lmao2](https://user-images.githubusercontent.com/111479268/224276404-f37600bb-3d0c-42ff-a9ac-954e8bdb0d0c.png)

**Comment by ianis58** [comment]
You seem to be using the script correctly.
Maybe it's related to the other issue where some people said the install.wim was not present and instead they have an install.esd file in the iso under "/sources/install.wim".
Please check if you have one of these two files in the sources directory in your E: drive.
If you have .esd, it's an issue because the current .bat tools doesn't support this (but there is a pull request that patch this issue).

Anyways I find it odd that people no longer have install.wim in their iso. I always download my windows iso from Microsoft directly, not from uup, and I always had install.wim file.
Anyone know where those iso with .esd come from?

**Comment by clorophilla** [comment]
Actually, there was a time (I think it was back in the Windows 8 era and the first 2 or 3 releases of 10) when Microsoft would send you an install.esd file inside its ISO file to make it as small as possible.

**Comment by elasticskyx** [comment]
Hi @MysticMCO 
I faced the same issue and modified the .BAT file to find the install.esd file instead of install.wim
You can download it here:
`https://github.com/ntdevlabs/tiny11builder/issues/18#issuecomment-1456896584`

**Comment by elasticskyx** [comment]
> You seem to be using the script correctly. Maybe it's related to the other issue where some people said the install.wim was not present and instead they have an install.esd file in the iso under "/sources/install.wim". Please check if you have one of these two files in the sources directory in your E: drive. If you have .esd, it's an issue because the current .bat tools doesn't support this (but there is a pull request that patch this issue).
> 
> Anyways I find it odd that people no longer have install.wim in their iso. I always download my windows iso from Microsoft directly, not from uup, and I always had install.wim file. Anyone know where those iso with .esd come from?

@ianis58 
I got the ISO from Microsoft website and found the install.wim missing
more details here: [https://github.com/ntdevlabs/tiny11builder/issues/18](url)

**Comment by ianis58** [comment]
@elasticskyx hi, thanks for your feedback about wim/esd.
I've seen your pull request and that's why I was able to answer this issue in the first place :)
Anyways I don't get why I never had esd in iso downloaded from Microsoft.
I've downloaded it multiple times (at least 5 or 6 times) between the 4 and 11 march 2023.

On the download page, I select the multi-edition.
Then on the language I select English United States.

I think I've seen esd recently in the already built Tiny11 builds.
